plugin/storage/cassandra/schema/create.sh

```diff
@@ -35,6 +35,9 @@ if [[ "$template" == "" ]]; then
         4)
             template=$(dirname $0)/v004.cql.tmpl
             ;;
+        5)
+            template=$(dirname $0)/v005.cql.tmpl
+            ;;
```

Comment:
Do we want to drop schema generation for v3?

Reply:
These are not Cassandra versions but our db schema evolutions. Does something need to change for Cassandra v5?

Reply:
no i just copied the same file to maintain the separation - let me know if you want me to combine it

Reply:
If the db schema is not changing there's no need to create a new file.

Reply:
should i just use the v4 file then?

Reply:
Yes - schema version and Cassandra version are not coupled.